tests/test_timefuncs.py

```diff
@@ -652,7 +653,12 @@ def test_types_to_numpy() -> None:
     td_s = pd.to_timedelta(pd.Series([10, 20]), "minutes")
     check(assert_type(td_s.to_numpy(), np.ndarray), np.ndarray)
     check(assert_type(td_s.to_numpy(dtype="int", copy=True), np.ndarray), np.ndarray)
-    check(assert_type(td_s.to_numpy(na_value=0), np.ndarray), np.ndarray)
+    if PD_LTE_15:
+        check(assert_type(td_s.to_numpy(na_value=0), np.ndarray), np.ndarray)
+    else:
+        check(
+            assert_type(td_s.to_numpy(na_value=pd.Timedelta(0)), np.ndarray), np.ndarray
```

Comment:
Can we always use the `else` case?

Reply:
> Can we always use the `else` case?

I tried that, and it didn't work, but now I tried again, and it did work!  Must have done something wrong the first time.

Fixed in commit [#c93d157](https://github.com/pandas-dev/pandas-stubs/pull/500/commits/c9ed1574d78ca4252b41c880a4f05dddc8f16d58)